pkg/client/cache/reflector.go

```diff
@@ -227,7 +227,10 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	resyncCh, cleanup := r.resyncChan()
 	defer cleanup()
 
-	options := unversioned.ListOptions{}
+	// Explicitly set "0" as resource version, as we can handle some
+	// delays in the reflector framework (those will be caught by
```

Comment:
Howe about this one:

Explicitly set "0" as resource version - it's fine for the List() to be served from cache and potentially be delayed relative to etcd contents. Reflector framework will catch up via Watch() anyway.

?